src/Tickets/Emails/Email_Abstract.php

```diff
@@ -362,4 +364,23 @@ public function get_additional_content(): string {
 	public function get_post() {
 		return get_page_by_path( $this->id, OBJECT, Email_Handler::POSTTYPE );
 	}
+
+	/**
+	 * Get edit URL.
+	 * 
+	 * @since TBD
+	 * 
+	 * @return string
+	 */
+	public function get_edit_url() {
+		// Force the `emails` tab.
+		$args = [
+			'tab'                    => Emails_Tab::$slug,
+			'section'                => Emails_Tab::$edit_section_slug,
+			Emails_Tab::$key_edit_id => $this->id,
```

Comment:
I think we can abstract things and just make it work with `section` as we do with Tickets Commerce and the payment gateways; there's no need to make an extra check if an existing email is being edited separately.